.changeset/yellow-dogs-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-defaults': patch
+---
+
+Use zero for empty response `contentLength` instead of printing `undefined` on the default log message.
```

Comment:
Could skip changeset tbh and just piggyback on the previous one, this is a fix for unreleased code anyway